Assets/ScriptableObjects/Items/DdrBody.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   id: DDR
   displayName: Dancer
   secretName: groovin'
-  displayDescription: Walk to the beat to reload!
+  displayDescription: Good stats, but you play a rthythm minigame to reload
```

Comment:
I think the only really notable stat change is the fire rate, so we could maybe write "good fire rate".

Maybe write "but you must **walk** to a rhythm minigame to reload" 

Also typo for rhythm

Reply:
Fix one typo and another appears 😔 

High feels more precise than good when it comes to this stat.
At this point I hope it is clear enough that people actively look for that mysterious rhythm minigame.